app/views/home/index.slim

```diff
@@ -11,7 +11,7 @@ h1 New Vasjuky Project
   br/
   = link_to('Logout', destroy_user_session_path, :method => :delete)
 
-- if current_user && current_user.admin?
+- if current_user && (current_user.admin? || current_user.moderator?)
```

Comment:
you shoud check it with pundit
https://github.com/elabs/pundit#headless-policies